src/SdkCommon/TestFramework/Microsoft.Azure.Test.HttpRecorder/Microsoft.Azure.Test.HttpRecorder.csproj

```diff
@@ -3,13 +3,13 @@
   <PropertyGroup>
     <Description>Microsoft.Azure.Test.HttpRecorder</Description>
     <AssemblyTitle>HttpRecorder Library for recording Clinet/Server communication in Azure</AssemblyTitle>
-    <VersionPrefix>1.8.0</VersionPrefix>
+    <VersionPrefix>1.8.1</VersionPrefix>
```

Comment:
@hovsepm version 1.8.1 just got release due to PS requirement of newtonsoft.json 6.0.8
This will have to released as 1.8.2, but from a different branch.
Retargeting the new branch.